src/dune_engine/action.ml

```diff
@@ -327,7 +327,45 @@ let is_useful_to distribute memoize =
     | true -> Maybe
     | false -> Clearly_not
 
-let is_useful_to_sandbox = is_useful_to false false
+let is_useful_to_sandbox =
```

Comment:
Since we're not using `is_useful_to false false` anymore, its first parameter is a constant `true`, so let's inline it!